### ISSUE TITLE
nullpointer exception occuring while try to import an api

### ISSUE THREAD

**Issue by celsoagra** [Axway-API-Management-Plus/apim-cli#390]
### APIM-CLI version

1.13.6

### API-Management version

7.7.20220228

### Bug description

The problem occurs when there's no "summary" field into swagger file.
![2023-04-28_17-34](https://user-images.githubusercontent.com/2061452/235249281-85e526d3-c75e-4a54-9952-88fbb63e11fb.png)

There's no info about which field should be mandatory on swagger file, so I suggest add some validation/info in these fields to permit a better undestanding of any error

https://github.com/Axway-API-Management-Plus/apim-cli/blob/746a7d46143f2a9869460d7baf86cee4dc683797/modules/apim-adapter/src/main/java/com/axway/apim/api/model/APIMethod.java#L129

### Steps to reproduce

generate some swagger file without summary field

### Relevant log output

```shell
2023-04-28 17:33:51,530 [APIImportApp] ERROR: null
java.lang.NullPointerException
	at com.axway.apim.api.model.APIMethod.equals(APIMethod.java:129)
	at java.util.ArrayList.indexOf(ArrayList.java:323)
	at java.util.ArrayList.contains(ArrayList.java:306)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566)
	at com.axway.apim.apiimport.actions.ManageApiMethods.updateApiMethods(ManageApiMethods.java:25)
	at com.axway.apim.apiimport.actions.CreateNewAPI.execute(CreateNewAPI.java:54)
	at com.axway.apim.apiimport.APIImportManager.applyChanges(APIImportManager.java:48)
	at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:95)
	at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.axway.apim.cli.APIManagerCLI.run(APIManagerCLI.java:133)
	at com.axway.apim.cli.APIManagerCLI.main(APIManagerCLI.java:77)
```

**Comment by rathnapandi** [comment]
Hi @celsoagra, 

We can treat summary as optional field as openapi specification does not say it is optional or mandatory.   we will fix it as part of next release. 

<h4 id="operation-object" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 0.5rem; font-family: &quot;Open Sans&quot;, sans-serif; font-weight: 500; line-height: 2.3125rem; color: rgb(33, 37, 41); font-size: 1.4375rem; letter-spacing: 0.005em; -webkit-font-smoothing: antialiased; padding-top: 30px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Operation Object</h4><p style="box-sizing: border-box; margin: 15px 0px; font-size: 16px; line-height: 24px; color: rgb(33, 37, 41); font-family: &quot;Open Sans&quot;, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Describes a single API operation on a path.</p><h5 id="fixed-fields-8" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 0.5rem; font-family: &quot;Open Sans&quot;, sans-serif; font-weight: 500; line-height: 2rem; color: rgb(33, 37, 41); font-size: 1.1875rem; letter-spacing: 0.005em; -webkit-font-smoothing: antialiased; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Fixed Fields</h5>

Field Name | Type | Description
-- | -- | --
tags | [string] | A list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier.
summary | string | A short summary of what the operation does.
description | string | A verbose explanation of the operation behavior. CommonMark syntax MAY be used for rich text representation.

Operation Object
Describes a single API operation on a path.

Fixed Fields
Field Name	Type	Description
tags	[string]	A list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier.
summary	string	A short summary of what the operation does.
description	string	A verbose explanation of the operation behavior. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.

**Comment by rathnapandi** [comment]
Fixed in v1.14.1
